Rakefile

```diff
@@ -1,6 +1,14 @@
+# frozen_string_literal: true
+
 # Add your own tasks in files placed in lib/tasks ending in .rake,
 # for example lib/tasks/capistrano.rake, and they will automatically be available to Rake.
 
 require_relative 'config/application'
 
 Rails.application.load_tasks
+
+# clear the default task injected by rspec
+task(:default).clear
+
+# and replace it with our own
+task default: [:ci]
```

Comment:
just do `task default: [:rubocop, :rspec]`

Reply:
Ok, running directly in travis with this instead of a rake task.

Reply:
👍